src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -161,6 +163,10 @@ public class ServerStartup : StartupBase
     {
         public override void ConfigureServices(IServiceCollection services)
         {
+            services.Configure<IdentityOptions>(options =>
+            {
+                options.ClaimsIdentity.UserNameClaimType = Claims.Name;
```

Comment:
It's worth noting it won't only affect access tokens but also regular ASP.NET Identity application cookies.
Instead, we may want to do the mapping in the authorization controller (see how it's done for the "sub" claim).

Reply:
done